packages/now-static-build/src/utils/_shared.ts

```diff
@@ -1,6 +1,18 @@
-import { PackageJson } from '@vercel/build-utils';
 import { constants, PathLike, promises as fs } from 'fs';
-import * as path from 'path';
+import { FileBlob } from '@vercel/build-utils';
+import { Files } from '@vercel/build-utils';
+import { Lambda } from '@vercel/build-utils';
+import { PackageJson } from '@vercel/build-utils';
```

Comment:
These 4 can be a single `import` statement

Reply:
True, I just don't like multiline imports 😛 
but I can change it